priv/catalog/373DB8.yaml

```diff
@@ -0,0 +1,74 @@
+id: "373DB8"
+name: fencing timeout
+group: Pacemaker
+description: |
+  Cluster fencing timeout is configured correctly in the cluster properties 'cib-bootstrap-options': stonith-timeout
+remediation: |
+  ## Abstract
+  The fencing timeout (`stonith-timeout`) determines the time Pacemaker will wait for fencing to succeed.
+  The recommended values on Azure are `144` seconds for SBD only or `900` seconds when using SBD combined with the Azure Fence agent.
+
+  ## Remediation
+  Execute the following command to adjust the timeout for your usecase:
+  ```crm configure property stonith-timeout=144```
+  or
+  ```crm configure property stonith-timeout=900```
+
+  ## References
+  Azure:
+
+    - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker#create-a-stonith-device-on-the-pacemaker-cluster
+
+  AWS:
+
+    - https://docs.aws.amazon.com/sap/latest/sap-hana/sap-hana-on-aws-cluster-resources.html
+
+  GCP:
+
+    - https://cloud.google.com/solutions/sap/docs/sap-hana-ha-config-sles#configure_the_general_cluster_properties
+
+  Nutanix:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-cluster-bootstrap-and-more
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-cluster-bootstrap-and-more
+
+facts:
+  - name: crm_config_properties
+    gatherer: cibadmin
+    argument: cib.configuration.crm_config.cluster_property_set
+  - name: resources_primitives
+    gatherer: cibadmin
+    argument: cib.configuration.resources.primitive
+
+values:
+  - name: expected_fencing_timeout
+    default: 150
+    conditions:
+      - value: 600
+        when: env.provider == "aws"
+      - value: 300
+        when: env.provider == "gcp"
+      - value: 144
+        when: env.provider == "azure"
+
+expectations:
+  - name: expectations_fencing_timeout
+    expect: |
+      let fencing_timeout=
+          facts.crm_config_properties
+               .find(|item| item.id == "cib-bootstrap-options").nvpair
+               .find(|prop| prop.name == "stonith-timeout").value;
+
+      let fence_azure_arm_detected=
+          facts.resources_primitives
+               .filter(|item| item.type == "fence_azure_arm").len() != 0;
+
+      if fence_azure_arm_detected {
+        fencing_timeout == 900;
+      } else {
+        fencing_timeout >= values.expected_fencing_timeout;
+      }
+
```

Comment:
Can we remove this final trailing new line?

Reply:
sure.